experiments/conductor/branches-all.yaml

```diff
@@ -20825,8 +20698,8 @@ branches:
       proto-service: ""
       proto-msg: google.cloud.resourcemanager.v3.Folder
       host-name: ""
-      notes: []
-      apis-enabled: []
+      notes:
+        - Terraform BETA
```

Comment:
Kind: should be "Folder" (https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/crds/resourcemanager_v1beta1_folder.yaml#L32)
That is I believe a different entry in branches-all.yaml

Reply:
Not sure I understand. The kind is `Folder` in this entry.